src/FormGroup.js

```diff
@@ -1,44 +1,107 @@
-import React from 'react';
 import classNames from 'classnames';
+import React from 'react';
+import deprecated from 'react-prop-types/lib/deprecated';
+
+import { Sizes } from './styleMaps';
+import bootstrapUtils, { bsClass, bsSizes } from './utils/bootstrapUtils';
+import ValidComponentChildren from './utils/ValidComponentChildren';
+
+const propTypes = {
+  /**
+   * Sets `id` on `<FormControl>` and `htmlFor` on `<FormGroup.Label>`.
+   */
+  controlId: React.PropTypes.string,
+  /**
+   * @private
+   */
+  standalone: deprecated(
+    React.PropTypes.bool,
+    'Use a `<FormControl>` or `<InputGroup>` directly.'
+  ),
+  validationState: React.PropTypes.oneOf(['success', 'warning', 'error']),
```

Comment:
does `bsStyle` make more sense as the prop name? that's in keeping with Button anyway...I guess `error != danger` though.